sdk/storage/azure-storage-blob-stress/scenarios-matrix.yaml

```diff
@@ -5,7 +5,7 @@ matrix:
       testScenario: downloadtofile
       sync: true
       sizeBytes: 1024
-      parallelRuns: 20
+      parallelRuns: 1
```

Comment:
Were we planning to decrease the parallel runs for all the test classes that use `maxConcurrency`? Or is this going to be updated in a follow up PR? I think it might be worth updating as part of this PR to avoid adding more load to tests than it already is at since by default we are setting `maxConcurrency` to `10` and making no changes to `parallelRuns` for most test cases. Mainly I want to avoid having these tests start failing due to resource constraints in the environment.